keyboards/mwstudio/mw65_rgb/rules.mk

```diff
@@ -16,6 +16,11 @@ NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
+
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = WS2812
+
 ENCODER_ENABLE = yes
+
+LAYOUTS = 65_ansi_blocker
```

Comment:
This won't actually work – [the `65_ansi_blocker` community layout has 67 keys](https://github.com/qmk/qmk_firmware/blob/7c186ebb974b4bbadbb74a7610bab5d2f66d030e/layouts/default/65_ansi_blocker/default_65_ansi_blocker/keymap.c#L17-L23), but the layout macro has 69, due to the assignments for the rotation of the rotary encoder.

I'd prefer the layout macro name remain as `LAYOUT` as well.

```suggestion
```

Reply:
Done, thanks